modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/message/QueryStartRequest.java

```diff
@@ -54,4 +54,9 @@ public interface QueryStartRequest extends ExecutionContextAwareMessage {
      */
     @Marshallable
     TxAttributes txAttributes();
+
+    /**
+     * Return last schema version, just a stub, need to be removed after IGNITE-18733.
+     */
+    long lastVersion();
```

Comment:
I believe it's better to rename to `schemaVersion`. Firstly, it's not last version, but version used to prepare the query. Secondly, this field should not be removed after IGNITE-18733, because general rule remains the same: the schema should not be changed during transaction's lifespan

Reply:
done